recipes/lue/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "0.2.0" %}
+
+
+package:
+  name: lue
+  version: {{ version }}
+
+
+source:
+  url: http://pcraster.geo.uu.nl/pcraster/packages/src/lue-{{ version }}.tar.bz2
+  sha256: 69926ca7d0fb81f70616958b007c749f925b56a80b98c32ac4b3e6860a5dd2ba
+
+
+build:
+  number: 0
+
+  missing_dso_whitelist:
+    - C:/Windows/System32/MSVCP140.dll             # [win]
+    - C:/Windows/System32/VCRUNTIME140.dll         # [win]
+    - C:/Windows/System32/OPENGL32.dll             # [win]
+    - Library/bin/MSVCP140.dll                     # [win]
+    - Library/bin/VCRUNTIME140.dll                 # [win]
+    - $RPATH/lue_py_data_model.cp37-win_amd64.pyd  # [win]
+    - $RPATH/lue_py_data_model.cp36-win_amd64.pyd  # [win]
+    - $RPATH/lue_py_data_model.cp38-win_amd64.pyd  # [win]
```

Comment:
Hmmm shouldn't these be part of the package? Why is it failing to find the link?

Reply:
We create a module (lue) and submodule (lue_py_data_model) from C++. Both files are installed in Python's site-package directory, and at runtime Python is able to load the lue and subsequently lue_py_data_model. That's what is done and tested in the run_test.py script, so this really works ;-)
Adding the lue_py_data_model.cp3x-win_amd64.pyd files to missing_dso_whitelist is not really pretty but was the only way I could figure out to pass the post-build tests done by conda-build. If you have a better suggestion we are interested to hear.

Reply:
Maybe you can have them be installed into the normal LIB directory and have lue look there?

Reply:
But it seems like `lue_py_data_model` is a python extension so that isn't right. You shouldn't need to have them on the whitelist.  Other packages do it this way. What is the error you were seeing exactly?

Reply:
yes, lue_py_data_model is a python extension. the error message is (shortened)
```   ...
   INFO (lue,Lib\site-packages\lue.cp36-win_amd64.pyd): Needed DSO api-ms-win-crt-stdio-l1-1-0.dll found in conda-forge::vs2015_runtime-14.16.27012-h30e32a0_2
   INFO (lue,Lib\site-packages\lue.cp36-win_amd64.pyd): Needed DSO C:/Windows\System32\VCRUNTIME140.dll found in the whitelist
  ERROR (lue,Lib\site-packages\lue.cp36-win_amd64.pyd): $RPATH\lue_py_data_model.cp36-win_amd64.pyd not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?
   INFO (lue,Lib\site-packages\lue.cp36-win_amd64.pyd): Needed DSO api-ms-win-crt-heap-l1-1-0.dll found in conda-forge::vs2015_runtime-14.16.27012-h30e32a0_2
   ...
overlinking check failed
['  ERROR (lue,Lib\\site-packages\\lue.cp36-win_amd64.pyd): $RPATH\\lue_py_data_model.cp36-win_amd64.pyd not found in sysroot, is this binary repackaging? .. do you need to use install_name_tool/patchelf?']
```
so I think the check is not right in this case, shared dependencies (so, dll) of a pyd should be tested and flagged if not found, but not dependent compiled Python modules (ie submodules). Finding those is done properly by Python if I understood it correctly

Reply:
We solved this by installing the so/pyd in different locations, lue in site-packages and lue_py_data_model in lib/bin; no more whitelisting of our submodule is required